src/back/auth/src/repo/repository.hpp

```diff
@@ -22,14 +25,13 @@ class Repository final : public components::LoggableComponentBase {
 		const components::ComponentConfig& conf,
 		const components::ComponentContext& ctx);
 
-	void SaveState(
-		const std::string& state,
-		const std::string& redirectUrl);
-
-	std::string TakeState(const std::string& state);
+	svetit::auth::table::State& State();
+	svetit::auth::table::Session& Session();
 
 private:
 	storages::postgres::ClusterPtr _pg;
+	svetit::auth::table::State _state;
+	svetit::auth::table::Session _session;
```

Comment:
Полный namespace можно не указывать:
```suggestion
	table::State& State();
	table::Session& Session();

private:
	storages::postgres::ClusterPtr _pg;
	table::State _state;
	table::Session _session;
```

Reply:
поправил везде